open-reac/src/main/resources/openreac/acopf.run

```diff
@@ -47,7 +47,7 @@ let tempstr := ctime();
 printf{LOG_KNITRO} "\n######################################################################\n";
 printf{LOG_KNITRO} "** ACopf solve: start (%s)\n\n",ctime();
 
-option knitro_options ("opttol=1 opttolabs=1e-1 feastol=1 feastolabs=1e-3 maxit=300 outlev=3");
+option knitro_options ("opttol=1 opttolabs=1e-1 feastol=1 feastolabs=1e-3 maxit=1000 outlev=3");
```

Comment:
Is it 500 or 1000 iterations? 

I was also wondering if there is an interest in documenting those knitro_options in the readme.md...